pattern_import_export_xlsx/tests/test_pattern_import.py

```diff
@@ -195,3 +207,11 @@ def disable_test_import_users_fail(self):
         self.assertEqual(ws["A1"].value, "#Error")
         self.assertTrue(ws["A2"].value)
         self.assertTrue(ws["A3"].value)
+
+    def test_import_partners_with_parents(self):
+        self._load_file("example.partners.parent.xlsx", self.ir_export_partner)
+        self.env.clear()
```

Comment:
why to you use self.env.clear ?

Reply:
Done. I presumed it was needed after seeing it once. Removed useless instances of it.